### PR TITLE
Fat 5517 create brands crud

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp/Controllers/BrandsController.cs [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        private IMapper config;
+
+        public BrandsController()
+        {
+            brandService = new BrandService();
+            config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<BrandDTO, BrandViewModel>();
+                cfg.CreateMap<BrandViewModel, BrandDTO>();
+            }).CreateMapper();
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/BrandsController.cs:22]: We can use new mapper to decrease the number of lines

File: StoreApp/Views/Brands/DetailsBrand.cshtml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿@model StoreApp.Models.Brands.BrandViewModel
+
+@{
+    ViewBag.Title = "Details brand";
+}
+
+<h3>Details @Model.Name</h3>
+
+<div>
+
```
Comment (AleksandroBoko) [inline StoreApp/Views/Brands/DetailsBrand.cshtml]: Btw it is not necessary to split the blocks by using space

Comment (Activict) [reply StoreApp/Views/Brands/DetailsBrand.cshtml]: fixed

File: StoreApp/Controllers/BrandsController.cs (hunk 1 above)
Comment (Activict) [reply StoreApp/Controllers/BrandsController.cs:22]: after discussion, left as is

**Pull request merged**